features/lwipstack/lwip/src/apps/altcp_tls/altcp_tls_mbedtls.c

```diff
@@ -98,7 +99,13 @@ extern const struct altcp_functions altcp_mbedtls_functions;
 struct altcp_tls_config {
   mbedtls_ssl_config conf;
   mbedtls_entropy_context entropy;
-  mbedtls_ctr_drbg_context ctr_drbg;
+#if defined(MBEDTLS_CTR_DRBG_C)
+  mbedtls_ctr_drbg_context    _drbg;
+#elif defined(MBEDTLS_HMAC_DRBG_C)
+  mbedtls_hmac_drbg_context   _drbg;
+#else
+#error "CTR or HMAC must be defined for coap_security_handler!"
```

Comment:
Why not always use the new Mbed OS global RNG and avoid use of DRBG entirely? The code should be much simplified if we always use the global RNG.

Reply:
We should run in non-Mbed OS, too?

Reply:
This is code in Mbed OS, though.